src/main/Osu.go

```diff
@@ -0,0 +1,107 @@
+package main
+
+import (
+	"github.com/thehowl/go-osuapi"
+	"github.com/bwmarrin/discordgo"
+	"fmt"
+	"strings"
+	"strconv"
+)
+
+func checkBeatmapLink(s *discordgo.Session, m *discordgo.MessageCreate){
+
+	//check if message contains only one beatmap link
+	if strings.Contains(m.Content, "https://osu.ppy.sh/s/") && strings.Contains(m.Content, "https://osu.ppy.sh/b/") || strings.Count(m.Content, "https://osu.ppy.sh/s/") > 1 || strings.Count(m.Content, "https://osu.ppy.sh/b/") > 1{
```

Comment:
An even better way of doing this would be 

`var countS := strings.Count(m.Content, "https://osu.ppy.sh/s/")`
`var countB := strings.Count(m.Content, "https://osu.ppy.sh/b/")`


And then later you can check for the one that has ` == 1 `, this would minimize looping over messages, since with a lot of messages this can take quite some time.